podvm/Dockerfile.podvm_builder

```diff
@@ -8,11 +8,18 @@
 #
 FROM ubuntu:20.04
 
-ARG GO_VERSION
-ARG PROTOC_VERSION
-ARG RUST_VERSION
-ARG YQ_VERSION
-ARG YQ_CHECKSUM
+ARG ARCH="amd64"
```

Comment:
Maybe consider the ARCH usage to be aligned with [Dockerfile.podvm_builder.rhel](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/podvm/Dockerfile.podvm_builder.rhel) (or to align the rhel one with this )